resources/views/admin/users/create.blade.php

```diff
@@ -0,0 +1,29 @@
+@extends('layouts.admin')
+@section('title', 'Create user')
+@section('content')
+<h1>Create a new user</h1>
+<div class="card">
+	<div class="card-body">
+		<div class="padded content">
+			@if ($errors->any())
+			<ul class="alert alert-danger">
+				@foreach ($errors->all() as $error)
+				<li>{{ $error }}</li>
+				@endforeach
+			</ul>
+			@endif
+
+			<form method="POST" action="{{ url('/admin/users') }}" accept-charset="UTF-8" enctype="multipart/form-data">
+				@csrf
+
+				@include('admin.users.form')
+				<div class="mt-3">
+					<input class="btn btn-primary" type="submit" value="Create">
+					<a href="{{ url('/admin/users') }}" title="Back" class="btn btn-outline-primary">Back</a>
```

Comment:
creo que hay varios tags `<a>` en estos documentos que en realidad son buttons. ¿podemos mejor usar el tag de `<button>`?

Reply:
Might do well to point them out specifically. Este de aquí, por ejemplo, es un link.

Reply:
si es un link por qué las classes dicen "btn"? tanto en el view como en el código se ve como un botón que redirecciona

Reply:
Sólo agregan styling

Reply:
resolving this comment. :)